fw/aux_port.h

```diff
@@ -322,6 +335,23 @@ class AuxPort {
       }
     }
 
+    if (!cui_amt22_ && cui_amt22_options_) {
+      // not sure how long the startup delay is for the amt22 but
```

Comment:
cappunc (specifically capitalization and lack of subject): https://google.github.io/styleguide/cppguide.html#Punctuation_Spelling_and_Grammar

The datasheet claims the startup time is 200ms.  You should at least reference that in the comment.  Unless you have some reason to believe it is actually faster than that, we should probably just use that value here.